.github/workflows/pre-release.yml

```diff
@@ -46,8 +43,16 @@ jobs:
       - name: Reveal Metabase ${{ matrix.edition }} properties
         id: version-properties
         run: |
-          echo "version=$(grep -o '^tag=.*' ./resources/version.properties | cut -d'=' -f2)" >> $GITHUB_OUTPUT
+          cat ./resources/version.properties
```

Comment:
I originally used to display the contents of `version.properties` only for my own sanity check, but then decided to leave it. It doesn't hurt, and it might help for future debugging if things go south.